### PR TITLE
Update Ecology and Evolutionary Biology Major (EEB)

### PR DESCRIPTION
**Pull request by richardhuangz**

### CODE REVIEW

File: tigerpath/majors_and_certificates/majors/EEB.json [hunk 1]
```diff
@@ -48,10 +48,31 @@
...
               "min_needed": 1,
               "explanation": null,
-              "course_list": [
-                "EEB 214/MOL 214",
-                "EEB 215/MOL 215"
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": 1,
+                  "min_needed": 1,
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/EEB.json:55]: https://eeb.princeton.edu/undergraduate/plan-study-0

"ISC Curriculum: Students who have completed the first full year of ISC are credited for MOL 214, CHM 201/2, and PHY 101/2. The student must take EEB 211. Students who complete 2 years of ISC can submit a short essay after reading Serengeti Rules and Selfish Gene to opt out of EEB 211."

File: tigerpath/majors_and_certificates/majors/EEB.json [hunk 2]
```diff
@@ -98,6 +119,30 @@
                 "CHM 215"
               ]
             },
+            {
```
Comment (churichard) [inline tigerpath/majors_and_certificates/majors/EEB.json:122]: I think you can simplify this by setting `max_counted` to 2 in one of the branches, and removing the other ISC branch.

File: tigerpath/majors_and_certificates/majors/EEB.json [hunk 3]
```diff
@@ -117,13 +162,34 @@
...
+              "explanation": null,
+              "course_list": [
+                "ISC 231",
+                "ISC 232",
+                "ISC 233",
+                "ISC 234"
+              ]
+            }
           ]
+
```
Comment (churichard) [inline tigerpath/majors_and_certificates/majors/EEB.json]: We don't need this blank line here

File: tigerpath/majors_and_certificates/majors/EEB.json (hunk 2 above)
Comment (richardhuangz) [reply tigerpath/majors_and_certificates/majors/EEB.json:122]: Gotcha, fixed.

**Review by churichard — APPROVED** [review]

**Pull request merged**